documentation/modules/managing/con-producer-config-properties.adoc

```diff
@@ -55,24 +56,51 @@ acks=all <1>
 ----
 
 <1> `acks=all` forces a partition leader to replicate messages to a certain number of followers before
-acknowledging that the message request was successfully received.
+acknowledging that the message request was successfully received. 
 
 The `acks=all` setting offers the strongest guarantee of delivery, but it will increase the latency between the producer sending a message and receiving acknowledgment.
 If you don't require such strong guarantees, a setting of `acks=0` or `acks=1` provides either no delivery guarantees or only acknowledgment that the leader partition has written the record to its log.
 
-The number of brokers which need to have appended the messages to their logs before the acknowledgment is sent to the producer is determined by the topic's `min.insync.replicas` configuration.
-A typical starting point is to have a topic replication factor of 3, with two in-sync replicas on other brokers.
-In this configuration, the producer can continue unaffected if a single broker is unavailable.
-If a second broker becomes unavailable, the producer won’t receive acknowledgments and won’t be able to produce more messages.
+With `acks=all`, the leader waits for all in-sync replicas to acknowledge message delivery.
+A topic's `min.insync.replicas` configuration sets the minimum required number of in-sync replica acknowledgements.
+The number of acknowledgements include that of the leader and followers. 
 
-.Topic configuration to support `acks=all`
-[source,env]
-----
-# ...
-min.insync.replicas=2 <1>
-# ...
-----
-<1> Use `2` in-sync replicas. The default is `1`.
+A typical starting point is to use the following configuration:
+
+* Producer
+** `acks=all` (default)
+* Broker/Topic:
+** `default.replication.factor=3` (default = `1`)
+** `min.insync.replicas=2` (default = `1`)
```

Comment:
I wonder if the `default = 1` would be a bit confusing. Most of our examples actually set the default to 2 and 3 already. E.g. in https://github.com/strimzi/strimzi-kafka-operator/blob/main/examples/kafka/kafka-persistent.yaml#L22-L23. It also doesn't really cover the `Broker/Topic` relationship which IMHO needs more explanation if you want to cover it both.

Reply:
I think it's okay to have them, to show that they are not default.
But I've also added this after: _Strimzi uses this configuration in the example configuration files for multi-node deployment of Kafka._